google-sheets/assets/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Get started with your Twilio Functions!</title>
+
+        <link rel="icon" href="https://twilio-labs.github.io/function-templates/static/v1/favicon.ico">
+        <link rel="stylesheet" href="https://twilio-labs.github.io/function-templates/static/v1/ce-paste-theme.css">
+        <script>
+         async function showGoogleSheetsStatus() {
+             const message = document.getElementById('status-message');
```

Comment:
While kind of obvious to us now that the id `status-message` is intrinsically linked here, its possible someone in the future might remove/change the name, thus breaking this functionality.

We might consider writing a test to ensure the name is present

Reply:
I want us to have a good test suite for these across the repo, but I think it'll require us to merge the Cypress testing branch in order to have those tests execute in a browser context.

Reply:
ok if we aren't writing a test for it now, you'll need to guard this. Since its possible in the future for `const message = document.getElementById('status-message');` to be undefined. All subsequent calls on `message.` could cause runtime errors.

So a simple check of `if (message) { ...} else { <error>}` should suffice